health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/Module.js

```
@@ -19,18 +29,51 @@
 
-
-  return (
-    <ProviderContext>
-      <EmployeeApp path={path} stateCode={stateCode} userType={userType} tenants={tenants}/>
-    </ProviderContext>
+  const { isLoading: isBoundaryOrderLoading, data: boundaryOrder } = Digit.Hooks.useCustomMDMS(
+    Digit.ULBService.getCurrentTenantId(),
+    "HCM",
+    [{ name: "BOUNDARYTYPES" }],
+    {
+      cacheTime: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)